app/src/main/java/org/oppia/app/administratorcontrols/AdministratorControlsActivityPresenter.kt

```diff
@@ -15,15 +19,34 @@ class AdministratorControlsActivityPresenter @Inject constructor(
   private val activity: AppCompatActivity
 ) {
   private lateinit var navigationDrawerFragment: NavigationDrawerFragment
+  private var isMultipane = false
 
-  fun handleOnCreate() {
+  fun handleOnCreate(extraControlsTitle: String?, lastLoadedFragment: String) {
     activity.setContentView(R.layout.administrator_controls_activity)
     setUpNavigationDrawer()
-    if (getAdministratorControlsFragment() == null) {
-      activity.supportFragmentManager.beginTransaction().add(
-        R.id.administrator_controls_fragment_placeholder,
-        AdministratorControlsFragment()
-      ).commitNow()
+    val titleTextView =
+      activity.findViewById<TextView>(R.id.extra_controls_title)
```

Comment:
I think you should control this into a data binding by introducing `<layout>` in xml file.

Reply:
Done